content/en/ecosystem/integrations.md

```diff
@@ -19,8 +28,7 @@ The entry must include the following:
 - Link to the documentation that explains how enable observability using
   OpenTelemetry
 
-Note that this list is for libraries, services, and apps that integrate
-OpenTelemetry **natively**.
+{{% alert title="Note" color="info" %}}
 
 If you provide external integration of OpenTelemetry for any kind of library,
 service, or app, then [add it to the registry](/ecosystem/registry/adding).
```

Comment:
rather than "then add it" maybe use "then consider adding it"?

Reply:
```suggestion
service, or app, then [consider adding it to the registry](/ecosystem/registry/adding).
```